docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,47 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 10 May 2023
+
+Frontend version: 1.0.0-20230509.1043 | Backend version: 1.61.9-230508.0923
```

Comment:
i see the dev backend version is 1.61.9-230508.0922

Reply:
@kpslp can trouble u to update per jerome's value

Reply:
changed